### PR TITLE
Add test notebooks and try to add tests to github actions

### PR DESCRIPTION
**Pull request by ajlee21**
This PR adds analysis notebooks, found in `Human_tests` and `Pseudomonas_tests`, to test using ponyo. These analysis scripts have already been reviewed during the review process of https://github.com/greenelab/simulate-expression-compendia. No need to look too deeply into these.

The main change to review is the addition of github workflows. This PR tried to add running these notebooks as github actions but they failed due to issues finding rpy2 path installation. For now, I have commented out those pytest lines in the github workflow to have a record.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          python-version: 3.8
+
+      - name: Setup conda environment
+        run: |
+          # $CONDA is an environment variable pointing to the root of the miniconda directory
+          $CONDA/bin/conda env update --file environment.yml --name base
+          $CONDA/bin/pip install .
+          $CONDA/bin/pip install pytest
+          $CONDA/bin/pip install nbval
+      #    export LD_LIBRARY_PATH=$(python -m rpy2.situation LD_LIBRARY_PATH):${LD_LIBRARY_PATH}
```
Comment (ben-heil) [inline .github/workflows/tests.yaml:26]: Is Github Actions failing to find rpy2 or the R installation itself? 

If it's the latter you may be able to manually specify the R_HOME variable with
``` bash 
R_HOME=`R RHOME`
```

Comment (ben-heil) [reply .github/workflows/tests.yaml:26]: Or maybe 
``` bash
R_HOME=`$CONDA/bin/R RHOME`
```
because github actions paths are a nightmare

Comment (ajlee21) [reply .github/workflows/tests.yaml:27]: The error message I was getting was `ValueError: r_home is None. Try python -m rpy2.situation` which I assumed meant that it wasn't finding R. 

Thank you for your suggestions, I tried them but they sadly didnt' work.

**Comment by ajlee21** [comment]
Thank you for looking into this. Perhaps I will return to this in the future..it would be really nice to get this working. But it is very confusing to me how to get the paths working in this case.

In terms of the linter, this is a limitation is due to this branch being on my forked repo: https://github.com/greenelab/ponyo/pull/3#issuecomment-645670468

**Pull request merged**